### ISSUE TITLE
Validator Version 1.1.6 Unstake Error 

### ISSUE THREAD

**Issue by wyzwanie** [Shardeum/shardeum-bug-reporting#266]
##### What is the issue?

:red_circle:  Cannot unstake. :red_circle: 
    
#### What impact does the issue have?

:red_circle: It is both visual bug as there is no feedback for user and system fail, as unstake does not work.. :red_circle: 

##### How to reproduce the issue? 

:red_circle: 
Click unstake in webUI with force toggle off
:red_circle: 

##### What other resources can you share regarding this issue?
screen from console
![Screenshot from 2023-03-29 19-10-40](https://user-images.githubusercontent.com/59451058/228616519-95c4f692-dcc4-4a74-9d8f-b38b8ec811b7.png)

**Comment by MarcusWentz** [comment]
Can you transfer SHM between wallets as a test?

**Comment by wyzwanie** [comment]
Yes. I have tried sending between wallets and it failed. What is strange it that I've input 3SHM as value to transfer and after fail i get attached, indicating transaction value was 0.
![Screenshot from 2023-03-29 21-24-33](https://user-images.githubusercontent.com/59451058/228646589-5058f32d-6a38-46b2-b422-889a73e418a1.png)

I was staking node today using one of those wallets and it took multiple tries (finally went through) with no information on frontend that something went wrong. in the console i've noticed error reports from axios library that looked like p2p notifications (i assume p2p as port was 9001) about transaction that timed out - not sure if that helps.

let me know if i can be of any more assistance.

**Comment by wyzwanie** [comment]
now after multiple retries it went through: https://explorer-sphinx.shardeum.org/tx/0xbebd89008218668f6df4549b85ce460b925db999abc76f1a25352694df4bf68c

i've also noticed that when it failed, recent transaction adress was shown as 0x00000000000....01

![Screenshot from 2023-03-29 21-39-59](https://user-images.githubusercontent.com/59451058/228649592-4b4a10a5-4298-417b-94df-16d036ddade8.png)

**Comment by MarcusWentz** [comment]
Q1. Were you able to stake as well?

Q2. What is your wallet address (the transaction link isn't loading right now)?

**Comment by wyzwanie** [comment]
> Q1. Were you able to stake as well?
> Q2. What is your wallet address (the transaction link isn't loading right now)?

RE1.
Yes I was able to stake eventually, but it took multiple retries, without any information on frontend.
RE2. 
from: 0x7279553ABF75424d31772aE816c8aB1bE75aBac2
to: 0x339c3c59E1Cf3998f5ABC14fF3D0930f292Ab019
amount: 3SHM

-----
what is strange is explorer doesnt show transaction and shows outdated wallet balance, while metamask shows up to date balance, check screenshot.
![Screenshot from 2023-03-29 21-47-57](https://user-images.githubusercontent.com/59451058/228651616-af636750-e42f-4b96-8f58-678a16354858.png)

**Comment by MarcusWentz** [comment]
The Explorer API for the transaction:

https://explorer-sphinx.shardeum.org/api/transaction?txHash=0xbebd89008218668f6df4549b85ce460b925db999abc76f1a25352694df4bf68c

shows the transaction is not found:
```json
{
  "success": false,
  "error": "This transaction is not found!"
}
```
The SHM transferred and you see the staked balance on the Validator GUI?

**Comment by wyzwanie** [comment]
Maybe we misunderstood. My understanding was that i was supposed to transfer tokens between wallets. I've done so using metamask wallet. Like i've mentioned transaction failed multiple times, with results shown above. Then it has completed given me above TX id. I can see that explorer api does not return anything. That is why im concerned. As wallets on metamask show balance after transfer, and indicate completed transaction. yet Explorer is empty and both wallets have balance prior to transaction.

current balance in metamask attached - see that balance in exploerer is different than in metamask and wallet address is the same

EDIT screenshot:
![Screenshot from 2023-03-29 22-04-50](https://user-images.githubusercontent.com/59451058/228655709-8ef2ecc0-b88b-48b0-86f3-53c4415f0bdb.png)
![Screenshot from 2023-03-29 22-04-11](https://user-images.githubusercontent.com/59451058/228655713-6d54698e-69db-4479-98a4-510dd886724b.png)

**Comment by wyzwanie** [comment]
im running nodes that are staked by those 2 wallet addresses and both stakes (eventually after multiple retries) worked fine and are registed in explorer and show in node GUI correctly.

My problem started when I wanted to test unstake, then you asked me to try transfering between wallets.

**Comment by MarcusWentz** [comment]
It looks like the test to transfer worked since the balances changed, but the Explorer hasn't updated yet.

Were you able to unstake?

**Comment by wyzwanie** [comment]
> It looks like the test to transfer worked since the balances changed, but the Explorer hasn't updated yet.
> 
> Are you still unable to unstake?

let me try again.

**Comment by wyzwanie** [comment]
no. i've tried 3 times and get as attached all the time and no information on front end.
![Screenshot from 2023-03-29 22-13-19](https://user-images.githubusercontent.com/59451058/228656757-bdb28868-59cb-44e8-8a49-4ccf83ded64f.png)

**Comment by wyzwanie** [comment]
> It looks like the test to transfer worked since the balances changed, but the Explorer hasn't updated yet.

So Explorer is not reflecting blockchain? If transaction went through and balances changed, how transaction can be not in the blockchain??

**Comment by MarcusWentz** [comment]
Seems related to this general transaction issue with nonce values not synced:
https://github.com/Shardeum/shardeum-bug-reporting/issues/249

**Comment by wyzwanie** [comment]
> Seems related to this general transaction issue with nonce values not synced: #249

if you need javascript developers, let me know where i can contribute.

**Comment by wyzwanie** [comment]
after upgrading to latest validator node version 1.1.6
i was able to successfully unstake nodes.
two notes:
1. when unstaking node that has pending payout it fails without any notification on front end, while in console there is clear message, about no payout time set or sth like that.
2. it is very worrysome that successful transaction between two addresses on same sphinx 1.x network using metamask wallet, does not show in transaction explorer.

**Comment by beatstick** [comment]
Was there any progress?

I cannot unstake either after updating to  1.17 (I forgot to unstake before updating).

[ethjs-query] while formatting outputs from RPC '{"value":{"code":-32603,"data":{"request":{"id":3491716565614,"jsonrpc":"2.0","params"[:...],"method":"eth_sendRawTransaction"}}}}'

I've tried from the console, from the dahsboard with and without the -force flag at least 20 times.
